helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -337,6 +337,10 @@ buffer:
   retry_max_times: 5
   retry_type: periodic
 
+# set to true to keep the structure created by docker or journald
+sendRaw:
```

Comment:
can we use `sendAllMetadata` instead? and make it flat? 
```
sendAllMetadata: true
```

Reply:
could you also update configmap template with this new name and boolean condition accordingly?